SimTKcommon/tests/CMakeLists.txt

```diff
@@ -26,27 +26,45 @@ foreach(TEST_PROG ${REGR_TESTS})
     get_filename_component(TEST_ROOT ${TEST_PROG} NAME_WE)
 
     if(BUILD_TESTS_AND_EXAMPLES_SHARED)
-        # Link with shared library
-        add_executable(${TEST_ROOT} ${TEST_PROG})
-        set_target_properties(${TEST_ROOT}
-        PROPERTIES
-          PROJECT_LABEL "Test_Regr - ${TEST_ROOT}")
-        target_link_libraries(${TEST_ROOT} 
-                    ${TEST_SHARED_TARGET})
-        add_test(${TEST_ROOT} ${EXECUTABLE_OUTPUT_PATH}/${TEST_ROOT})
+        if (NOT ${TEST_ROOT} MATCHES "TestAutodiffADOLCCommon")
```

Comment:
This seems like a hack for testing -- we'll need a better way to decide when to link examples to the adolc libraries. I'm OK with having this here for now, but it needs a "TODO" and an explanatory comment saying what it does and explaining the limitations and promising a to repair later. Or if I'm misinterpreting, the comment should clarify why this is right.

Reply:
@sherm1, I have added comments where I used `if (NOT <> MATCHES <> )`. I will try to come up with a better way in another PR (I have added TODOs).